### PR TITLE
Response to suggestions in proofreading of traditional vaccines

### PR DESCRIPTION
**Pull request by rando2**
This is for the traditional vaccines only -- I still have a couple more points to address, but this should make most of the suggested modifications.

### CODE REVIEW

**Comment by agitter** [comment]
Accepting changes in https://github.com/greenelab/covid19-review/pull/1168 created merge conflicts here (which I resolved), so I'll stop editing https://github.com/greenelab/covid19-review/pull/1168 until we finish and merge this.

**Comment by rando2** [comment]
Thanks Tony! I'll move the efficacy results back into the main discussion of the vaccine development this afternoon and then tag you for review.

File: content/22.vaccines-trad.md [hunk 1]
```diff
@@ -328,9 +326,16 @@ In a phase I/II trial, a two-dose regimen of NVX-CoV2373 was found to induce ant
...
+Additionally, in both trials, the vaccine was found to be well-tolerated [@doi:10.1101/2021.10.05.21264567; @doi:10.1056/NEJMoa2107659].
+The conclusions of both trials indicate that the NVX-CoV2373 vaccine is safe and effective against COVID-19, and those who received this vaccine through the trials were considered fully vaccinated prior to the vaccine receiving an EUA [@url:https://ir.novavax.com/Novavax-Statement-on-UK-and-Mexico-Phase-3-Clinical-Trial-Participants-Considered-Fully-Vaccinated-in-the-US].
+The Novavax vaccine has since been authorized for use in several places, including the United Kingdom [@url:https://www.gov.uk/government/news/novavax-covid-19-vaccine-nuvaxovid-approved-by-mhra], the E.U. [@url:https://www.ema.europa.eu/en/news/ema-recommends-nuvaxovid-authorisation-eu], and the U.S.A [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-emergency-use-novavax-covid-19-vaccine-adjuvanted].
+<!-- Need to indicate it is EUA? What have we been doing elsewhere? Deleted the line below about the June FDA vote, the actual EUA didn't come until July. -->.
```
Comment (rando2) [inline content/22.vaccines-trad.md:338]: I think this is OK now with the language changed to "authorized" -- does that sound OK?

**Comment by rando2** [comment]
@agitter OK, I think this now addresses the restructure-related comments! I will not plan to push any more changes here for now.

File: content/22.vaccines-trad.md (hunk 1 above)
Comment (agitter) [reply content/22.vaccines-trad.md:338]: Yes, the authorized language is more precise and eliminated any possible confusion.

**Review by agitter — APPROVED** [review]
Thanks for all the edits in response to my comments.  Let's merge this so we can continue working on https://github.com/greenelab/covid19-review/pull/1168.

**Pull request merged**